config/additional_links.json

```
@@ -0,0 +1,17 @@
+[
+    {
+        "displayName":"Sample link 1",
+        "url":"http://sample-link-1.com",
+        "icon": "supervisor_account"
+    },
+    {
+        "displayName":"Sample link 2",
+        "url":"http://sample-link-2.com",
+        "icon": "speaker_notes"
+    },
+    {
+        "displayName":"Sample link 3",
+        "url":"http://sample-link-3.com",
+        "openInNewTab": false
+    }
+]
```

following https://github.com/camicroscope/caMicroscope/commit/9291074f2d274dcded4e9938800c04df7e0cc379

```suggestion
[
    {
        "displayName":"Bug Report",
        "url":"https://goo.gl/forms/mgyhx4ADH0UuEQJ53",
        "icon": "bug_report"
    }
]
```

---

@birm Shouldn't it contain an example case where `openInNewTab` is set to `false`, for clarity?

---

we could set it to true, which should still show off the field

---

Alright, I'll do that